core/src/main/java/org/apache/iceberg/hadoop/HadoopCatalog.java

```diff
@@ -19,122 +19,116 @@
 
 package org.apache.iceberg.hadoop;
 
-import com.google.common.base.Joiner;
 import com.google.common.base.Preconditions;
 import java.io.Closeable;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
 import java.util.Map;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
 import org.apache.iceberg.BaseMetastoreCatalog;
 import org.apache.iceberg.PartitionSpec;
 import org.apache.iceberg.Schema;
+import org.apache.iceberg.Table;
 import org.apache.iceberg.TableMetadata;
 import org.apache.iceberg.TableOperations;
 import org.apache.iceberg.catalog.TableIdentifier;
-import org.apache.iceberg.exceptions.AlreadyExistsException;
 import org.apache.iceberg.exceptions.RuntimeIOException;
 
 /**
- * The HadoopCatalog provides a way to manage the path-based table just like catalog. It uses
- * a specified directory under a specified filesystem as the warehouse directory, and organizes
+ * HadoopCatalog provides a way to use table names like db.table to work with path-based tables under a common
+ * location. It uses a specified directory under a specified filesystem as the warehouse directory, and organizes
  * two levels directories that mapped to the database and the table respectively. The HadoopCatalog
- * takes a URI as its root directory, and creates a child directory "iceberg/warehouse" as the
- * warehouse directory. When creating a table such as $db.$tbl, it creates $db.db/$tbl under
+ * takes a URI as the warehouse directory. When creating a table such as $db.$tbl, it creates $db/$tbl under
  * iceberg/warehouse, and put the table metadata into the directory.
  *
- * The HadoopCatalog now supports createtable, droptable, the renametable is not supported yet.
+ * The HadoopCatalog now supports {@link org.apache.iceberg.catalog.Catalog#createTable},
+ * {@link org.apache.iceberg.catalog.Catalog#dropTable}, the {@link org.apache.iceberg.catalog.Catalog#renameTable}
```

Comment:
These can be referenced without the class name, like this: `{@link #createTable}`

Reply:
If I refer it without full qualified class name, Intellij shows the symbol is not found. Maybe because these APIs are from another package/jar (api).

Reply:
I see, it's referencing the interface's method. This should be fine.